docs/api/types.rst

```diff
@@ -12,6 +12,8 @@ The table below shows the correspondence between EdgeDB and JavaScript types.
 +----------------------+-----------------------------------------------------+
 | EdgeDB Type          |  JavaScript Type                                    |
 +======================+=====================================================+
+| ``Set``              | ``Set``                                             |
```

Comment:
This isn't correct; the corresponding JS type is Array

Reply:
changed